Bigtable/src/ResumableStream.php

```diff
@@ -116,6 +121,17 @@ public function __construct(
         $this->callOptions['retrySettings'] = [
             'retriesEnabled' => false
         ];
+
+        $this->delayFunction = function (int $backoffCount) {
+            $initialDelayMillis = 100;
+            $initialDelayMultiplier = 1.3;
+            $maxDelayMillis = 60000;
```

Comment:
I copied these values from the retry parameter defaults

Reply:
please align with the values here:

https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java#L135-L137

Reply:
Looks like max attempts also differs from ours (they have `setMaxAttempts(10)`, we are using `3`). Should we change this default to be `10` for `readRows` as well?

Reply:
Discussed offline - we are keeping max attempts at 3, but also resetting attempts after every successful row